crates/weaver_resolver/data/registry-test-15-attr-any/registry/registry-messaging.yaml

```diff
@@ -0,0 +1,19 @@
+groups:
+  - id: registry.test
+    type: attribute_group
+    brief: 'Used for testing any and template[any]'
+    attributes:
+      - id: test.any
+        stability: stable
+        type: any
```

Comment:
we'll probably need to filter out `any` attributes in semconv jinja templates on Java

https://github.com/open-telemetry/semantic-conventions-java/blob/3c7edf34f5b548e64a63793e3772da40754d874c/buildscripts/templates/registry/incubating_java/IncubatingSemanticAttributes.java.j2#L10.

Not sure if there are other languages that have similar API and won't be able to define a constant for `any` attribute (yet).

Reply:
Created https://github.com/open-telemetry/semantic-conventions-java/issues/208 and https://github.com/open-telemetry/opentelemetry-go/issues/6691 to give maintainers a heads up